docs/user-guide.md

```diff
@@ -90,13 +88,42 @@ be too. You might use this method like this:
 Tools could then use the `@Nullable` information to determine that the first use
 is safe but the second is not.
 
+As far as JSpecify is concerned, `String` and `@Nullable String` are _different_
+types. A variable of type `String` can reference any string. A variable
+of type `@Nullable String` can too, but it can also be null. This means that
+`String` is a _subtype_ of `@Nullable String`, in the same way that `Integer` is
```

Comment:
The comma isn't necessary here.

Reply:
I think it helps, though. If I were saying this aloud I would pause. I'm not saying "`String` is a subtype of `@Nullable String` in the same way that...", as opposed to some other way. I'm saying "`String` is a subtype of `@Nullable String`", and here's an example that shows what that means. However I notice that I say "in the same way" twice in rapid succession so some editing is needed. :-)

Reply:
Fair enough.